backend/ops_api/tests/ops/agreement/test_agreement.py

```diff
@@ -422,6 +422,22 @@ def test_agreements_patch_by_id_grant(auth_client, loaded_db):
     assert [m.id for m in agreement.team_members] == [1]
 
 
+@pytest.mark.usefixtures("app_ctx")
+def test_agreements_patch_by_id_just_notes(auth_client, loaded_db):
+    """PATCH with just notes to test out other fields being optional"""
+    response = auth_client.patch(
+        "/api/v1/agreements/1",
+        json={
+            "notes": "Test PATCH",
+        },
+    )
+    assert response.status_code == 200
+
+    stmt = select(Agreement).where(Agreement.id == 1)
+    agreement = loaded_db.scalar(stmt)
+    assert agreement.notes == "Test PATCH"
```

Comment:
As indicated yesterday, do we want to clean up this record post PATCH?

Reply:
I can, I just copied what Steve had done, and didn't do anything else with it.

Reply:
I can work on this (after this PR) in my branch.
It's not just this one test but all the PUT, PATCH tests that I was thinking of creating a new record and deleting when done.

Reply:
@stevtek You can use what I did in that test as an example to get similar fixes in others if you want ;)